android/accessibility_guide.md

```diff
@@ -24,6 +24,21 @@ Some development tips:
     - Generally let the system handle focus changes: it's unintuitive when it jumps around.
 - Overriding the accessibility APIs in code is a code smell: generally, you can make the changes you need using the methods recommended in the docs, which provides a more maintainable and intuitive experience to users. Also note the code APIs are unintuitive and not well documented.
 
+### Cut out redundant steps with `android:importantForAccessibility="no"`
+Unintuitively, setting some view to not be important for accessibility improves accessibility.
+For example, take this layout that reprents a menu item that consists of an image and text.
+```
+<LinearLayout>
+  <ImageView/>
+  <TextView/>
+</LinearLayout>
+```
+When a user swipes right past this composite view, the cursor will first land on the LinearLayout, and then on the TextView:
+<img alt="Animation of cursor landing menu items and their text children" src="https://github.com/mozilla-mobile/shared-docs/raw/master/android/images/double-nav.gif" width="240">
+
+This is both inefficient and confusing for the user, since they will encounter each item twice. To streamline this experience, set `android:importantForAccessibility="no"` on the `TextView`. This will tell TalkBack not to bother with that node:
```

Comment:
I feel like you can get this behavior by setting `android:focusable="true"` on the `LinearLayout` and that that is the recommended style in the Google accessibility guidelines: is that true? If so, what's the difference between this approach and the `focusable` approach?

Reply:
The `focusable` attribute implies implicitly that `importantForAccessibility` is true for that view.

I looked at the Google guide, and I'm not sure what they mean. containers that have `focusable="true"` will still let you step into their children with TalkBack. I learned this from trial and error mostly.

Reply:
This whole tip was inspired by an [issue](https://github.com/mozilla-mobile/android-components/pull/2237) in android components. In that case, the `LinearLayout` already had `android:focusable="true"`.

Reply:
@eeejay I see – maybe we should add a blurb about why "focusable="true"" doesn't work in cases like these (because others might remember similarly to me)? Then I think we're good to land.

Reply:
As we discussed elsewhere, this is only tangentially related to `focusable`, so I think mentioning it here is just confusing.